nextflow_schema.json

```diff
@@ -38,6 +38,100 @@
                 }
             }
         },
+        "database": {
+            "title": "Databases",
+            "type": "object",
+            "description": "Select databases to be run on all samples.",
+            "fa_icon": "fas fa-terminal",
+            "properties": {
+                "pointfinder_database": {
+                    "enum": [
+                        "Automatic Selection",
+                        "Enterococcus faecium",
+                        "Enterococcus faecalis",
+                        "Helicobacter pylori",
+                        "Salmonella",
+                        "Campylobacter",
+                        "Escherichia coli"
+                    ],
+                    "description": "Select a single Pointfinder database to use on all samples (overriding metadata option). Validated Organisms: Enterococcus faecium, Enterococcus faecalis, Helicobacter pylori, Salmonella, Campylobacter, Escherichia coli"
+                },
+                "plasmidfinder_database": {
+                    "enum": ["All", "gram_positive", "enterobacteriales"],
+                    "description": "The database type to use for plasmidfinder {gram_positive, enterobacteriales}. Defaults to using all available database types to search for plasmids. [All]."
+                },
+                "mlst_scheme": {
+                    "type": "string",
+                    "description": "Specify scheme name, visit https://github.com/tseemann/mlst/tree/master/db/pubmlst for supported scheme genera available. [None]",
+                    "default": "None"
```

Comment:
Could this be set to default to `"Automatic"`, where the `Automatic` string is then used by StarAMR to automatically select the MLST scheme.

Essentially, in StarAMR, the `[None]` string reported for the `--mlst-scheme` parameter help indicates the behaviour if it is unset (not that if it is set to the string `None`). The default behaviour is to automatically select an MLST scheme.

For the staramr pipeline, since parameters are shown to users in IRIDA Next, instead of showing the value `None`, I'd like to show the value `Automatic` so that it's descriptive of the behaviour (automatic selection of an MLST scheme).

Reply:
This was a misunderstanding on my part of how StarAMR worked. I thought it only used a MLST scheme if one was selected by the user based on the `staramr search -h` message
```
Specify scheme name, visit https://github.com/tseemann/mlst/tree/master/db/pubmlst for supported scheme genera available. [None]
```

Reply:
No problem. It is likely not well documented in staramr. Thank you for the change.